Resources/translations/SonataArticleBundle.en.xliff

```diff
@@ -30,6 +30,54 @@
                 <source>new.fragment</source>
                 <target>New fragment</target>
             </trans-unit>
+            <trans-unit id="breadcrumb.link_article_list">
+                <source>breadcrumb.link_article_list</source>
+                <target>Articles</target>
+            </trans-unit>
+            <trans-unit id="list.label_title">
+                <source>list.label_title</source>
+                <target>Title</target>
+            </trans-unit>
+            <trans-unit id="list.label_updated_at">
+                <source>list.label_updated_at</source>
+                <target>Updated At</target>
```

Comment:
Why do you capitalize all words? You don't even do it for all languages, please be consistent ("Updated at")